### DISCUSSION TITLE
[FEATURE]建议专业版的 Docker 代理配置与 1Panel 代理配置分离

### DISCUSSION THREAD

**Discussion by Anyexyz** [1Panel-dev/1Panel#7080] [Ideas]
### 1Panel 版本

v1.10.21-beta.1

### 请描述您的需求或者改进建议

> 设置代理服务器后，将在以下场景中生效：
> - 应用商店的安装包下载和同步（专业版功能）
> - 系统版本升级及获取更新说明（专业版功能）
> - Docker 的网络访问将通过代理服务器进行（专业版功能）
> - 系统许可证的验证和同步

希望可以将docker的代理配置与 1Panel 的代理配置分离，~~（正常来说这俩不该用同一个代理吧）~~

### 请描述你建议的实现方案

将 Docker 的代理配置移动到 `容器` - `配置` 下

**Comment by wanghe-fit2cloud** [comment]
感谢反馈！目前在设置代理服务器时，支持用户选择是否同步到 Docker 代理。
如果选择同步，代理服务器配置会自动写入 `daemon.json` 文件中。
若不选择同步，可在 `容器` - `配置` 页面手动编辑 `daemon.json`，此功能属于社区版。
> 若在 `容器` - `配置` 页面也增加代理服务器配置选项，可能会与面板设置页面功能重叠。
